src/main/scala/org/pac4j/http4s/DefaultHttpActionAdapter.scala

```diff
@@ -0,0 +1,21 @@
+package org.pac4j.http4s
+
+import org.http4s.{Response, Status}
+import org.pac4j.core.context.HttpConstants
+import org.pac4j.core.http.adapter.HttpActionAdapter
+import scalaz.concurrent.Task
+
+object DefaultHttpActionAdapter extends HttpActionAdapter[Task[Response], Http4sWebContext] {
```

Comment:
An more generally, I like to see a Javadoc on classes

Reply:
Done